File: snippets/proxy/create-session/create-session.cs
using System;
using Twilio;
using Twilio.Rest.Proxy.V1.Service;

// Find your Account Sid and Token at twilio.com/console
const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
const string authToken = "your_auth_token";

TwilioClient.Init(accountSid, authToken);

var session = SessionResource.Create(
    pathServiceSid: "KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
);

Console.WriteLine(session.Sid);

File: snippets/proxy/create-session/create-session.java
package proxy;

import com.twilio.Twilio;
import com.twilio.rest.proxy.v1.service.Session;

public class create_session {

    private static final String ACCOUNT_SID = System.getenv("ACCOUNT_SID");
    private static final String AUTH_TOKEN = System.getenv("AUTH_TOKEN");

    public static void main(String[] args) {
        Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

        // https://www.twilio.com/docs/proxy/api/session
        Session session = Session.creator(System.getenv("PROXY_SERVICE_SID"))
                .setUniqueName("MyFirstSession").create();
        System.out.println("+ New Proxy service session SID: " + session.getSid());
    }
}
